### PR TITLE
Add module.hot reload check in index.js

### PR DESCRIPTION
**Pull request by kenneth-bakke**

### CODE REVIEW

**Comment by kenneth-bakke** [comment]
Let me know what you guys think!

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -2,6 +2,6 @@ import React from 'react';
 
 export default function App() {
   return (
-    <div>Hello from App!!!</div>
+    <div>This is a shopping place</div>
```
Comment (kenneth-bakke) [inline client/src/components/App.jsx:5]: I felt like trying something with shopping

**Comment by Elliehunt8** [comment]
Looks good

**Review by yasinnkhann — CHANGES_REQUESTED** [review]
Not sure if it works yet.

**Pull request closed without merging**